application/values-test.yaml

```diff
@@ -223,6 +225,12 @@ deployment:
   # Graceful termination timeout
   terminationGracePeriodSeconds: 120
 
+  # Container lifecycle management for graceful startup and shutdown procedures
+  lifecycle:
```

Comment:
Why it is not inside `deployment`?

Reply:
@kseniyashaydurova 
It's in deployment
I think you were misled by the huge number of formatting errors in the source repository - vscode just went crazy and didn't show the nesting of variables correctly.  I've fixed it now and it's displaying correctly.
![Screenshot from 2024-05-28 14-37-31](https://github.com/saritasa-nest/stakater-application/assets/93929010/d632abbe-440f-4618-80c1-8411c8ee68f6)